RELEASE/scripts/sl_ascend.ash

```diff
@@ -6990,6 +6998,10 @@ boolean L11_unlockEd()
 	if(get_property("sl_tavern") != "finished")
 	{
 		print("Uh oh, didn\'t do the tavern and we are at the pyramid....", "red");
+
+		// Forcing Tavern. Will perform 1 ADV and then go back and do the tavern.
+		set_property("sl_forceTavern", true);
+		print("DEBUG - ForceTavern: "+ get_property("sl_forceTavern"));
```

Comment:
Why not `return L3_tavern()` instead of adventuring once here?

Reply:
If L3_Tavern() returned false wouldn't it then skip the Pyramid?

Reply:
Sure, make it `if (L3_Tavern()) return true;`

Reply:
Awesome, thank you. Fixed.